backend/service/chaos/experimentation/experimentstore/experiment_store.go

```diff
@@ -82,7 +82,7 @@ func (fs *experimentStore) CreateExperiments(ctx context.Context, experiments []
 		        experiment_config_id,
 		        execution_time,
 		        creation_time)
-            VALUES ($1, $2, tstzrange(NOW(), NULL), NOW())`
+            VALUES ($1, $2, tstzrange(NOW(), NULL, '[]'), NOW())`
```

Comment:
noob question - what is this `[]` for?

Reply:
this means that both sides of the range bound are inclusive